kalite/i18n/management/commands/makemessages.py

```diff
@@ -11,16 +11,27 @@
 from django.core.management.commands import makemessages
 
 
+IGNORE_PATTERNS = [
+    "*/node_modules*",
+    "*dist-packages*",
+    "*python-packages*",
+    "*/kalite/static*",
+    "*bundle*.js",
+    "*/ka-lite/build/*",
+    "*/js/i18n/*.js",
+    "*/ka-lite/docs/*"
```

Comment:
This ignore pattern is mainly for `gettext` not to see the text in RST's generated buttons and links. We need more integration for the docs itself.